tests/integration-tests/tests/schedulers/test_slurm.py

```diff
@@ -1890,9 +1902,9 @@ def _test_enable_fast_capacity_failover(
     )
 
 
-def _test_update_without_update_queue_params(pcluster_config_reader, cluster, remote_command_executor):
+def _test_update_without_update_queue_params(partition, pcluster_config_reader, cluster, remote_command_executor):
     """Test update without queue param change, clustermgtd and slurmctld not restart."""
-    updated_config_file = pcluster_config_reader(config_file="pcluster.config.update.yaml")
+    updated_config_file = pcluster_config_reader(config_file="pcluster.config.update.yaml", partition=partition)
```

Comment:
Instead of adding an additional parameter to every method that manipulates the cluster config file, let's create a fixture like the one used for region, that injects the `partition` parameter in every cluster config file

Reply:
Done